subsys/usb/usbc/Kconfig

```diff
@@ -0,0 +1,45 @@
+# USBC stack configuration options
+
+# Copyright (c) 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_COMPAT_USBC_PORT := usbc-port
+
+menuconfig USBC_STACK
+	bool "USBC Stack Support"
+	default y
+	depends on DT_HAS_USBC_PORT_ENABLED
```

Comment:
See comment above. It should not be possible to enable this high level flag otherwise than manually.
Remove these lines.

Reply:
Done